ib.py

```diff
@@ -170,6 +170,23 @@ def services_kill(args):
         print "Unknown service"
         sys.exit(1)
 
+def changelog_create(args):
+    # Build docker container
```

Comment:
In the doc above  you write to set the env  var CHANGELOG_GITHUB_TOKEN, but I don't see any reference in the code. is it used?